src/System-FileRegistry/ZipArchive.extension.st

```diff
@@ -1,5 +1,15 @@
 Extension { #name : 'ZipArchive' }
 
+{ #category : '*System-FileRegistry' }
+ZipArchive class >> extractAllIn: aFileReferenceOrFileName [
+	"Service method to extract all contents of a zip."
+	| directory |
+	directory := (UIManager default chooseDirectoryFrom: aFileReferenceOrFileName asFileReference) ifNil: [^ self].
```

Comment:
There is still a ref to UIManager. And I would remove it to keep 

```
^ (self new)
		readFrom: aFileReferenceOrFileName;
		extractAllTo: directory
```
and remove the UI popping behavior.

Reply:
Ok, that would remove the service to select where to extract. But I think it is not essential for UI

Reply:
Done